hist/hist/inc/TScatter.h

```diff
@@ -57,6 +57,8 @@ class TScatter : public TNamed, public TAttLine, public TAttFill, public TAttMar
    Double_t  GetMinMarkerSize() const {return fMinMarkerSize;} ///< Get the smallest marker size used to paint the markers
    TGraph   *GetGraph()  const {return fGraph;}                ///< Get the graph holding X and Y positions
    TH2F     *GetHistogram() const;                             ///< Get the graph histogram used for drawing axis
+   TAxis    *GetXaxis() const ;
```

Comment:
Why are these two functions needed?

Reply:
If it's for ease of use, then they should be three?  (GetZaxis  in addition) ?

Reply:
Yes , ease of use, GetZaxis is coming. The PR is not finished yet. I am looking at distancetoprimitive now, One cannot pick the axis....